gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1,6 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 1991, 2022 IBM Corp. and others
- *
+ * Copyright (c) 1991, 2023
```

Comment:
Why was `IBM Corp. and others` removed from the copyright? Seems accidental.

Reply:
Certainly not intentional. Thanks for spotting. Fixed.